### DISCUSSION TITLE
movement system

### DISCUSSION THREAD

**Discussion by grewqcxeqeq** [SmartlyDressedGames/Unturned-3.x-Community#4729] [Ideas]
Hi Nelson,

I’d love to see you update the movement system and add parkour mechanics and climbing features. It would be great to have rolls and other movement elements in the game so that modders could create higher-quality animations.

Currently, such features can only be achieved through workarounds and by replacing a lot of the game's code.

What do I mean by parkour? Things like creating animations for grabbing onto rooftops, moving while hanging from ledges, performing flips in place, and even adding shooting animations. I think you might like the idea!

Let me know if you'd like any adjustments to this!

There are already videos on YouTube showing parkour, but it still looks like a workaround:

https://www.youtube.com/watch?v=VPtIiJ8BbIc
https://www.youtube.com/watch?v=T9ZugT7mkNE&t=109s

I also think new game modes could be added, not just PVP or RP, but also things like spleef or maps where players need to strafe.

**Comment by WawaEgo** [comment]
cool idea

**Comment by SDGNelson** [comment]
It would be really interesting!
